src/managing-deis/platform-ssl.md

```diff
@@ -1,3 +1,62 @@
 # Platform SSL
 
-TODO (bacongobbler): rewrite for v2
+SSL/TLS is the standard security technology for establishing an encrypted link between a web server
+and a browser. This link ensures that all data passed between the web server and browsers remain
+private and integral.
+
+To enable SSL for your cluster and all apps running upon it, you can add an SSL key to your load
+balancer. You must either provide an SSL certificate that was registered with a CA or provide
+[your own self-signed SSL certificate](../reference-guide/creating_self_signed_ssl.md).
+
+## Installing SSL on a Load Balancer
+
+On most cloud-based load balancers, you can install a SSL certificate onto the load balancer
+itself. Any communication inbound to the load balancer will be encrypted while the internal
+components of Deis will still communicate over HTTP.
+
+When you install Deis, Kubernetes will provision a load balancer for the routers. To enable SSL,
+you will need to modify the listener settings on the load balancer:
```

Comment:
Although, the previous paragraph says "most cloud providers," so maybe you already covered your bases and narrowed the scope of the discussion.

Up to you if you want to make any further clarification here or not.  It need not stand in the way of my LGTM.